definitions/json/EventToComplexTypes/EventToComplexTypes.json

```diff
@@ -685,7 +685,7 @@
     "CaseEventID": "initiateCase",
     "CaseFieldID": "claimantHearingPreference",
     "ListElementCode": "hearing_preferences",
-    "EventElementLabel": "What are the claimant's hearing preferences",
+    "EventElementLabel": "What are the claimant's hearing preferences1",
```

Comment:
I assume the number at the end was for test purposes and can be removed?

Reply:
```suggestion
    "EventElementLabel": "What are the claimant's hearing preferences",
```